CHANGES_NEXT_RELEASE

```diff
@@ -19,6 +19,10 @@
 - Fix: Allowing trailing semicolon in a string filter, but only one (Issue #2086)
 - Fix: NGSIv2 geoquery syntax errors using 400 Bad Request instead of 422 Unprocessable Entity (according to rules defined in #1286)
 - Fix: NGSIv2 URI PATH must be all in lowercase, otherwise "service not found" (Issue #2057)
+- Fix: A bug fixed in URI param 'q', avoiding false matches when using string match operator (~=) that matches AND some operator that does not match.
+- Add: New URI param 'mq' for matching in metadata (Issue #1156) 
+- Add: New param 'mq' (both in URI param for list all entities operation and expression sub-field in subscriptions) for matching in metadata (#1156)
```

Comment:
Line 23 is redundant, already covered by lines 24. Suggestion: just remove line 23.